modules/36_buildings/simple/declarations.gms

```diff
@@ -8,7 +8,12 @@
 
 Parameters
   p36_CESMkup(ttot,all_regi,all_in)        "parameter for those CES markup cost accounted as investment cost in the budget [trUSD/CES input]"
+$call "awk -F ',' '$0 !~ /^\*/ { exit NF }' ./modules/36_buildings/simple/input/p36_floorspace_scen.cs4r"
+$ifthenE.resComFloor0 (errorLevel)eq(5)
+  p36_floorspace(tall,all_regi,secBuild36) "buildings floorspace, billion m2, in simple realization only used for reporting"
+$else.resComFloor0
   p36_floorspace(tall,all_regi)            "buildings floorspace, billion m2, in simple realization only used for reporting"
+$endif.resComFloor0
```

Comment:
From GAMS' point of view, this declaration is not necessary, since `./modules/36_buildings/simple/datainput.gms` contains a declaration.
Unless @LaviniaBaumstark/RSE have strong feelings towards declaring everything in `declarations.gms` _as well_ (in which case, there would be quite some code cleaning to be done), I would suggest to drop it and reduce the number of esoteric `awk` calls.